### PR TITLE
Rabbitmq queue metricset message rates

### PR DESCRIPTION
**Pull request by kvalev**
This pull requests extends the existing RabbitMQ queue metricset to include the message rates.

Related to #6442

### CODE REVIEW

File: libbeat/common/schema/mapstriface/mapstriface.go [hunk 1]
```diff
@@ -171,6 +171,11 @@ func Bool(key string, opts ...schema.SchemaOption) schema.Conv {
 	return schema.SetOptions(schema.Conv{Key: key, Func: toBool}, opts)
 }
 
+// Float creates a Conv object for parsing floats
+func Float(key string, opts ...schema.SchemaOption) schema.Conv {
+	return schema.SetOptions(schema.Conv{Key: key, Func: toInteger}, opts)
```
Comment (ruflin) [inline libbeat/common/schema/mapstriface/mapstriface.go]: You use the `toInteger` function here which I think does not return the expected result?

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -256,6 +255,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Making the MongoDB module GA. {pull}6554[6554]
 - Allow to disable labels `dedot` in Docker module, in favor of a safe way to keep dots. {pull}6490[6490]
 - Add experimental module to collect metrics from munin nodes. {pull}6517[6517]
+- Add connections metricset to RabbitMQ module {pull}6548[6548]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: This line should be removed for this PR.

Comment (kvalev) [reply CHANGELOG.asciidoc]: Sure, I can remove it, but currently this line is in the wrong section - Bugfixes, while it should be in Added.

Comment (kvalev) [reply CHANGELOG.asciidoc]: I left the line in the change log, let me know if I should revert it.

Comment (ruflin) [reply CHANGELOG.asciidoc]: I'm just realising now that this log line has been moved and not added.

+1 on moving it to the right section.

**Comment by jsoriano** [comment]
@kvalev branch needs rebase

**Comment by CodingSpiderFox** [comment]
@kvalev: any updates?

**Comment by jsoriano** [comment]
jenkins, test this  please

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -193,6 +193,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Added support for haproxy 1.7 and 1.8. {pull}6793[6793]
 - Add accumulated I/O stats to diskio in the line of `docker stats`. {pull}6701[6701]
 - Ignore virtual filesystem types by default in system module. {pull}6819[6819]
+- Add connections metricset to RabbitMQ module {pull}6548[6548]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: Was this line added on purpose? Was it missing in a previews PR? If yes, the PR number seems to be off.

**Comment by ruflin** [comment]
I resolved the changelog conflict myself. I'm kind of surprised the mapstriface change does not conflict.

**Comment by jsoriano** [comment]
jenkins, test this please

**Comment by CodingSpiderFox** [comment]
What else is needed here?

**Comment by ruflin** [comment]
@CodingSpiderFox Could you rebase this again on master?

**Comment by CodingSpiderFox** [comment]
I'm not good doing git stuff. Still learning. How can I do the rebase? Fork this repo, pull the branch kvalev and push it to my repo, rebase it on master and then open another PR?

**Comment by jsoriano** [comment]
@CodingSpiderFox yes, this is what you'd need to do if you want to follow with these changes :slightly_smiling_face:
To pull kvalev's branch you will need to add his repository as a different remote, ping us if you get stuck at some step.
In any case let's wait a little bit to see if @kvalev can rebase the branch again, if not maybe we can also finish it.

**Comment by jsoriano** [comment]
jenkins, test this again please

**Comment by jsoriano** [comment]
There is some issue with imports formatting.

**Comment by ruflin** [comment]
@CodingSpiderFox Sorry, my mistake. I somehow assumed you were the creator of the PR.

**Comment by CodingSpiderFox** [comment]
I need this feature very soon

**Comment by jsoriano** [comment]
Rebased and merged as #6964

**Pull request closed without merging**